coreos-base/coreos/coreos-0.0.1.ebuild

```diff
@@ -69,11 +69,9 @@ RDEPEND=">=sys-apps/baselayout-3.0.0"
 # If protocol 1 is installed it must be configured to provide the default
 # implementation based on whether protocol 2 is enabled or not.
 RDEPEND="${RDEPEND}
-	etcd_protocols_1? (
-		dev-db/etcd:1
-		!etcd_protocols_2? ( dev-db/etcdctl )
-	)
-	etcd_protocols_2? ( dev-db/etcd:2 )
+	dev-db/etcdctl
```

Comment:
No, that ebuild is the etcd0 version of etcdctl, you do not want it. we should remove it at this point.